src/cycle-dom.js

```diff
@@ -62,6 +63,15 @@ let CycleDOM = {
   h,
 
   /**
+   * shortcuts to [hyperscript-helpers](
+   * https://github.com/ohanhi/hyperscript-helpers).
+   * This is a helper for extending virtual-hyperscript. Create virtual DOM elements with `div({className: 'wrapper'}, [ h1('Header') ])` instead of `h('div.wrapper', [ h('h1', 'Header') ])`.
+   * @name hh
+   */
+
+  hh,
```

Comment:
This should be `...hh` not `hh`. But don't worry, I'll merge it and continue from here.